### DISCUSSION TITLE
PHP not being detected in a .php file in sample document example

### DISCUSSION THREAD

**Discussion by HelloMukama** [mpdf/mpdf#1613] [Q&A] [answered]
i am using the example https://github.com/mpdf/mpdf-examples/blob/master/example34_invoice_example.php and im replacing line 61 with the code below

```
<?php
$d = new DateTime('now');
$d->setTimezone(new DateTimeZone('UTC'));
$timeToday =  $d->format("Y-m-d H:i:s");

$html = '
...
60.     <div style="text-align: right">
61.         Date: <?php $timeToday ?> 
62.     </div>
...

```
but the code gets printed as is.
and the real date isn't printed out.
How can I solve this because I intend to make my table dynamic enough to display changing data from the database?

I tried the following format but no luck yet and I'm on a quick deliverable and therefore looking for a quick solution 

```
<?php
$d = new DateTime('now');
$d->setTimezone(new DateTimeZone('UTC'));
$timeToday =  $d->format("Y-m-d H:i:s");

$html = '
...

60.     <div style="text-align: right">
';  // $html  ends here

61. <?php Date: <?php $timeToday ?>  ?>

// continuation begins here...
$html += '

62.     </div>
...

```

**Comment by finwe** [comment, marked as answer]
This is not a templating language, just concatenate strings

```php
$html = '
<div style="text-align: right">
    Date: ' . $timeToday . '
</div>';
```

**Reply by HelloMukama** [reply]
this has worked thanks.

**Reply by HelloMukama** [reply]
though with further PHP expressions like loops and so, it still raises issues

**Reply by finwe** [reply]
Not an issue of mPDF.

**Reply by HelloMukama** [reply]
fixed it. It was a php version issue
